mu-plugins/global-fonts/index.php

```diff
@@ -141,44 +141,52 @@ function get_font_url( $font, $subset ) {
 	}
 
 	switch ( $lower_font ) {
+		case 'courier prime':
+			$font_folder    = 'Courier Prime/';
+			$font_file_name = 'CourierPrime-Regular';
+			break;
+		case 'courier prime bold':
+			$font_folder    = 'Courier Prime/';
+			$font_file_name = 'CourierPrime-Bold';
```

Comment:
These should end with a `-`. I believe the subset gets hyphenated somewhere downstream.